src/applications/static-pages/facilities/FacilityPatientSatisfactionScoresWidget.jsx

```diff
@@ -59,37 +129,8 @@ export default class FacilityPatientSatisfactionScoresWidget extends React.Compo
           they need care right away
         </p>
         <div className="usa-grid-full">
-          <div className="usa-width-one-half">
-            <div className="usa-width-one-half vads-u-background-color--gray-lightest vads-u-padding--1p5">
-              <p className="vads-u-margin--0">Primary care</p>
-              <p
-                id="facility-patient-satisfaction-scores-primary-urgent-score"
-                className="vads-u-font-size--lg vads-u-font-weight--bold vads-u-margin--0 vads-u-font-family--serif"
-              >
-                {this.state.facility.attributes.feedback.health
-                  .primaryCareUrgent
-                  ? displayPercent(
-                      this.state.facility.attributes.feedback.health
-                        .primaryCareUrgent,
-                    )
-                  : 'N/A'}
-              </p>
-            </div>
-            <div className="usa-width-one-half vads-u-background-color--gray-lightest vads-u-padding--1p5">
-              <p className="vads-u-margin--0">Specialty care</p>
-              <p
-                id="facility-patient-satisfaction-scores-specialty-urgent-score"
-                className="vads-u-font-size--lg vads-u-font-weight--bold vads-u-margin--0 vads-u-font-family--serif"
-              >
-                {this.state.facility.attributes.feedback.health
-                  .specialtyCareUrgent
-                  ? displayPercent(
-                      this.state.facility.attributes.feedback.health
-                        .specialtyCareUrgent,
-                    )
-                  : 'N/A'}
-              </p>
-            </div>
+          <div className="usa-width-one-half vads-u-display--flex">
+            {urgentScoreMarkup}
```

Comment:
Typically, we would do something like

```
{!!this.state.facility.attributes.feedback.health.primaryCareRoutine && 
  <div className="facility-satisfaction-tile vads-u-background-color--gray-lightest vads-u-padding--1p5 vads-u-margin-right--1">
      <p className="vads-u-margin--0">Primary care</p>
      <p
         id="facility-patient-satisfaction-scores-primary-urgent-score"
         className="vads-u-font-size--lg vads-u-font-weight--bold vads-u-margin--0 vads-u-font-family--serif"
       >
          {displayPercent(
            this.state.facility.attributes.feedback.health.primaryCareUrgent,
          )}
        </p>
    </div>
}
```

for these conditionals. That keeps the logic inline with the rest of the JSX and makes it a little easier to read, since you don't have to flip back and forth between here and the push statements.

Reply:
i will do these in my re-factor for 2271